recipes/rocksdb/all/conandata.yml

```diff
@@ -27,6 +30,10 @@ sources:
     url: "https://github.com/facebook/rocksdb/archive/refs/tags/v6.20.3.tar.gz"
     sha256: "c6502c7aae641b7e20fafa6c2b92273d935d2b7b2707135ebd9a67b092169dca"
 patches:
+  "9.6.1":
+    - patch_file: "patches/9.6.1-0001-support-clang.patch"
+      patch_description: "Fix build with gcc 13 by including cstdint"
```

Comment:
Hi @toge this is a bit suprising for me. the patch description makes it look like it will only be adding `#include <cstdin>` where necessary, but the diff here (and in the linked PR) seems to be quite bigger in scope than just that. Could you elaborate into what it ecxactly is doing? Thanks!

Reply:
@AbrilRBS 
Sorry for late response.
It's my mistake.

9.7.2 has been released which is included above patch.

Reply:
No worries, thanks for taking the time to circle back to this PR :)